coffee_lib/src/plugin.rs

```diff
@@ -104,6 +104,8 @@ pub struct Plugin {
     conf: Option<Conf>,
     /// FIXME: this field shouldn't be optional
     pub commit: Option<String>,
+    /// Optional for now to be backward compatible
```

Comment:
This is not useful as a doc comment, the doc comment should explain what the field is, and then add more context on why it is optional